x/data/server/msg_server.go

```diff
@@ -100,7 +100,7 @@ func (s serverImpl) SignData(goCtx context.Context, request *data.MsgSignDataReq
 
 		store.Set(key, emptyBz)
 		// set reverse lookup key
-		store.Set(SignerCIDKey(signer, cidStr), emptyBz)
+		store.Set(SignerCIDKey(signer, cidBz), emptyBz)
```

Comment:
Why is this changed here? Shouldn't we choose one way, and keep it consistent across `SignerCIDKey()` and `CIDSignerKey()`? Currently `CIDSignerKey()` still takes `cidStr string`...

Reply:
A string is not needed since this is the second part of the index key. It's an optimization. Maybe better to be consistent, but also good to optimize where possible no?

Reply:
Yes!